app/scripts/utils/use-pathname.ts

```diff
@@ -13,14 +13,18 @@ import { useEffect, useState } from 'react';
  */
 export const usePathname = () => {
   const [pathname, setPathname] = useState(
-    typeof window !== 'undefined' ? window.location.pathname : ''
+    typeof window !== 'undefined'
+      ? window.location.pathname.replace(process.env.PUBLIC_URL ?? '', '')
+      : ''
   );
 
   useEffect(() => {
     if (typeof window === 'undefined') return;
 
     const updatePathname = () => {
-      setPathname(window.location.pathname);
+      setPathname(
+        window.location.pathname.replace(process.env.PUBLIC_URL ?? '', '')
```

Comment:
Some of our instances configure public_url as a full url, (not only subpath - like https://earth.gov/ghgcenter) https://github.com/US-GHG-Center/veda-config-ghg/blob/develop/.github/workflows/deploy.yml#L82

Would this approach still work in this case?

Reply:
@hanbyul-here good catch, thanks. I'll push a fix for this case.